[2018-11-08 20:47:21] [    INFO] --- Setup logging. Log file: ./output/gt1/MNIST/no_cuda=True/08-11-18-20h-47m-21s/logs.txt (setup.py:531)
[2018-11-08 20:47:21] [    INFO] --- Arguments saved to: ./output/gt1/MNIST/no_cuda=True/08-11-18-20h-47m-21s/args.txt (setup.py:534)
[2018-11-08 20:47:21] [    INFO] --- Initialize Tensorboard SummaryWriter (setup.py:539)
[2018-11-08 20:47:23] [ WARNING] --- Git status is ignored! (RunMe.py:205)
[2018-11-08 20:47:23] [    INFO] --- Randomly chosen seed is: 3309764677 (setup.py:616)
[2018-11-08 20:47:23] [    INFO] --- Model CNN_basic expects input size of (32, 32) (my_task.py:85)
[2018-11-08 20:47:23] [   ERROR] --- Unhandled error: NameError("name 'classify' is not defined",) (RunMe.py:261)
[2018-11-08 20:47:23] [   ERROR] --- Traceback (most recent call last):
  File "./template/RunMe.py", line 254, in _execute
    **args.__dict__)
  File "/home/dusan/DeepDIVA/template/runner/my_task/my_task.py", line 89, in single_run
    classify=classify, **kwargs)
NameError: name 'classify' is not defined
 (RunMe.py:262)
[2018-11-08 20:47:23] [   ERROR] --- Execution finished with errors :( (RunMe.py:263)
